src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -698,6 +701,8 @@
           text: az aks create -g MyResourceGroup -n MyManagedCluster --enable-azure-service-mesh
         - name: Create a kubernetes cluster with Azure Monitor Metrics enabled.
           text: az aks create -g MyResourceGroup -n MyManagedCluster --enable-azuremonitormetrics
+        - name: Create a kubernetes cluster with Azure Monitor App Monitoring enabled
+          text: az aks create -g MyResourceGroup -n MyManagedCluster --enable-azure-monitor-app-monitoring
```

Comment:
I dont see preview API version. Did we test these changes for both create and update scenarios?

Reply:
Yes both the scenarios have been tested. There are live tests for the same. The default api version of the aks preview is the latest preview api